M2/Macaulay2/packages/MatrixSchubert/ASM_Lists.m2

```diff
@@ -0,0 +1,76 @@
+
+
+------------------------------
+-- INPUT: an integer n between 1 and 7
+-- OUTPUT: a list of matrices containing all ASMs of size n
+------------------------------
+
+ASMFullList = method()
+ASMFullList ZZ := List => (n) -> (
+    if (n < 1 or n > 7) then error("There is no available list for this n.");
+    filename := concatenate(
+	MatrixSchubert#"source directory",
+	"MatrixSchubert/ASMData/full/", toString n, ".txt");
```

Comment:
I was just going to suggest this before your new push:
```suggestion
    filename := concatenate(MatrixSchubert#"auxiliary files", "ASMData/full/", toString n, ".txt");
```
But I think this works the same. 👍